README.md

```diff
@@ -1,17 +1,71 @@
-# GA4GH service-info specification [![](https://travis-ci.org/ga4gh-discovery/service-info.svg?branch=develop)](https://travis-ci.org/ga4gh-discovery/service-info) [![](https://img.shields.io/badge/license-Apache%202-blue.svg)](https://raw.githubusercontent.com/ga4gh-discovery/service-info/develop/LICENSE)
+# GA4GH service-info specification v1.0.0 [![](https://travis-ci.org/ga4gh-discovery/service-info.svg?branch=develop)](https://travis-ci.org/ga4gh-discovery/service-info) [![](https://img.shields.io/badge/license-Apache%202-blue.svg)](https://raw.githubusercontent.com/ga4gh-discovery/service-info/develop/LICENSE)
```

Comment:
I'd suggest removing the version number from the title since this will be a living repo with a tag for a particular version.

Reply:
Agreed but I worry that the first question we could be asked for is what version is this. I'll remove though

Reply:
Done